src/sidebar/util/wrap-react-component.js

```diff
@@ -14,14 +18,16 @@ function useExpressionBinding(propName) {
  * has been created.
  */
 class ReactController {
-  constructor($element, $injector, $scope, type) {
+  constructor($element, services, $scope, type) {
     /** The DOM element where the React component should be rendered. */
     this.domElement = $element[0];
 
     /**
-     * The Angular service injector, used by this component and its descendants.
+     * The services injector, used by this component and its descendants.
+     *
+     * @type {Injector}
      */
-    this.$injector = $injector;
+    this.services = services;
```

Comment:
At this point, I'm starting to feel a little bit of cognitive stress from the differing vocabulary: `services`, `container`, `injector`. Can you help me understand by explaining the specific shades of semantic meaning?

Reply:
We probably need to revise the terminology. There are really only two distinct concepts here: The dependency injection container and the _things_ registered in it. Those things currently fall into two categories: 1) Services which provide methods to trigger API calls or other effects and 2) Objects whose values are determined on startup and need to be exposed by the app setup code (in `src/sidebar/index.js`) to the rest of the app.